config/application.rb

```diff
@@ -5,10 +5,7 @@
 require "active_model/railtie"
 require "active_job/railtie"
 require "active_record/railtie"
-require "active_storage/engine"
 require "action_controller/railtie"
-require "action_mailer/railtie"
-require "action_mailbox/engine"
 require "action_text/engine"
 require "action_view/railtie"
 require "action_cable/engine"
```

Comment:
can we remove action_cable and action_text?

Reply:
yes, will show in a upcomming pr